core/domain/domain.fr.yml

```diff
@@ -556,15 +547,13 @@ responses:
     - text: "Suis-je autorisé à aller à l'extérieur?"
 
   utter_qa_sample_stats:
-    - text:
-        "Où puis-je trouver les statistiques à jour sur les cas et les décès?"
+    - text: "Où puis-je trouver les statistiques à jour sur les cas et les décès?"
     - text: "Combien de cas à Montréal?"
     - text: "Combien de personnes sont mortes à ce jour?"
     - text: "Combien de cas au Québec?"
     - text: "Combien de personnes sont infectées au Québec?"
     - text: "Combien de cas sont aux États-Unis?"
-    - text:
-        "De la population infectée dans le monde entier quel sera le rapport des
+    - text: "De la population infectée dans le monde entier quel sera le rapport des
         personnes asymptomatiques?"
```

Comment:
Why were these messages reformatted?

Reply:
Not sure. I didn't use the good vscode formatting because I was at the root, but I just tried with the workspace settings and it does something different, but still corrects this.